h/static/scripts/annotator/guest.coffee

```diff
@@ -1,15 +1,18 @@
 Annotator = require('annotator')
 $ = Annotator.$
+Util = Annotator.Util
+selectionEndTimeout = null
 
 module.exports = class Guest extends Annotator
   SHOW_HIGHLIGHTS_CLASS = 'annotator-highlights-always-on'
 
   # Events to be bound on Annotator#element.
   events:
+    "setVisibleHighlights": "setVisibleHighlights"
     ".annotator-adder button click":     "onAdderClick"
     ".annotator-adder button mousedown": "onAdderMousedown"
     ".annotator-adder button mouseup":   "onAdderMouseup"
-    "setVisibleHighlights": "setVisibleHighlights"
+    "document onselectionchange": "userSelectionChanged"
```

Comment:
I don't _think_ this works because our typical injection target is the body and `document` is not a selector underneath the body, it's above it. You will have to do this in `_setupDocumentEvents` and then unbind it yourself in `destroy`.